.github/CONTRIBUTING.md

```diff
@@ -1,26 +1,35 @@
-Contribution guidelines
-We're happy to accept 3rd-party contributions. Please make sure you read this document before you do any work though, as we have some expectations related to the content and quality of change sets.
+# Contribution guidelines
```

Comment:
remove this file there's a central one 

https://github.com/hmcts/.github/blob/master/CONTRIBUTING.md